x-pack/test/security_solution_cypress/cypress/tasks/prebuilt_rules_preview.ts

```diff
@@ -227,8 +229,15 @@ export const assertCommonPropertiesShown = (properties: Partial<PrebuiltRuleAsse
   cy.get(INTERVAL_TITLE).should('have.text', 'Runs every');
   cy.get(INTERVAL_VALUE).should('contain.text', properties.interval);
 
-  cy.get(FROM_TITLE).should('have.text', 'Additional look-back time');
-  cy.get(FROM_VALUE).invoke('attr', 'data-test-subj').should('contain', properties.from);
+  cy.get(LOOK_BACK_TITLE).should('have.text', 'Additional look-back time');
+  cy.get(LOOK_BACK_VALUE)
+    .invoke('attr', 'data-test-subj')
+    .should(
+      'contain',
+      toLargestTimeDuration(
+        calcDateMathDiff(properties.from ?? 'now-6m', `now-${properties.interval}`) ?? 0
+      )
+    );
```

Comment:
I wouldn't expect tests for the rule preview feature to be changed by this PR. Was there a good reason? Please revert if not.

Reply:
The reason is in showing two types of rule schedule 1) simple rule schedule and 2) full rule schedule. The first one shows `lookback` but the latter shows `from` and `to`. Due to most probably legacy reasons `lookback`'s `data-test-subj` uses `from` value instead of `lookback`. It was changes in this scope of this PR since it touched rule schedule related components.